mods/auth/src/utils/auth_utils.ts

```diff
@@ -53,7 +53,9 @@ export default class AuthUtils {
     let accessTokenData: JwtPayload
     try {
       accessTokenData = await this.handler.decode(token.accessToken, privateKey)
```

Comment:
You could simplify by saying: return await this.handler.decode(token.accessToken, privateKey)